dashboard/app/models/script.rb

```diff
@@ -1381,7 +1382,8 @@ def summarize(include_stages = true, user = nil, include_bonus_levels = false)
       family_name: family_name,
       version_year: version_year,
       assigned_section_id: assigned_section_id,
-      hasStandards: has_standards_associations?
+      hasStandards: has_standards_associations?,
+      tts: tts?,
```

Comment:
I'm not sure this is the right syntax -- this checks if the field exists, not what it's value is. please check that this does what you expect with false, null, etc

Reply:
```
[development] dashboard > s = Script.new
=> #<Script id: nil, name: nil, created_at: nil, updated_at: nil, wrapup_video_id: nil, hidden: false, user_id: nil, login_required: false, properties: {}, new_name: nil, family_name: nil>
[development] dashboard > s.tts
=> nil
[development] dashboard > s.tts?
=> false
[development] dashboard > s.tts = false
=> false
[development] dashboard > s.tts?
=> false
[development] dashboard > s.tts = true
=> true
[development] dashboard > s.tts?
=> true
[development] dashboard > s.tts = "foo"
=> "foo"
[development] dashboard > s.tts?
=> true
```

I think it works because I added it to `serialized_attrs` as `tts` and not `has_tts`, would you prefer changing it?

Reply:
my mistake! if it's all working, it's fine with me as is.